packages/discussions/lib/loaders/Discussion.js

```diff
@@ -0,0 +1,29 @@
+const DataLoader = require('dataloader')
+
+module.exports = (context) => ({
+  byId: new DataLoader( ids =>
+    context.pgdb.public.discussions.find({ id: ids })
+  ),
+  Commenter: {
+    discussionPreferences: new DataLoader( async (keyObjs) => {
+      console.log('load discussionPreferences')
+      const discussionId = keyObjs[0].discussionId
```

Comment:
You shouldn't assume all `keyObjs` have the same discussion. Use a [or and](https://pogi.readthedocs.io/en/latest/API/condition/#and-or) condition:

```js
const condition = {or: keyObjs.map(o => ({
 and: o
}))}
```